config/locales/translations/zendesk_apps_support.yml

```diff
@@ -157,6 +165,14 @@ parts:
       key: "txt.apps.admin.error.app_build.style_in_template"
       title: "App builder job: <style> tags in template error"
       value: "<style> tag in %{template}. Use an app.css file instead."
+  - translation:
+      key: "txt.apps.admin.error.app_build.garden_cdn_reference"
+      title: "App builder job: Zendesk Garden CSS CDN reference in iframe template"
+      value: "Deprecated Zendesk Garden CSS referenced in %{file}. Update <link> to %{reference} to load from jsDelivr instead. See: https://jsdelivr.com/?query=zendeskgarden"
+  - translation:
+      key: "txt.apps.admin.error.app_build.legacy_sdk_url_reference"
+      title: "App builder job: legacy ZAF SDK url reference in iframe template"
+      value: "Deprecated Zendesk App Framework SDK referenced in %{file}. Update <script> for %{reference} to %{current_sdk_url}"
```

Comment:
I usually prefer to point to the docs and be less specific here, in case the details change, it's easier to change the docs.

Reply:
Mm I'm not sure I agree. It feels like a nicer developer experience to me that if we're rejecting an app because of an incorrect reference that we then make it as easy as possible to correct that error (by including the link to the file here). 

It's a small PR to update docs, but equally it's only a single line here: https://github.com/zendesk/zendesk_apps_support/pull/217/files#diff-0770dbea04431e199f0b2b1f37d34610R37, and realistically this isn't likely to change often

Reply:
Can you add screenshot please?